cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -170,8 +169,9 @@ object ScannerCommand : CommandWithHelp() {
         }
 
         val scanner = scannerFactory.create(config)
+        ScanResultsStorage.storage = LocalFileStorage(getUserOrtDirectory().resolve(TOOL_NAME))
```

Comment:
This should be done before the storage configuration is applied above, otherwise it overwrites the configured storage.

Reply:
Very true, thanks. Done.